pkg/gui/controllers/files_controller.go

```diff
@@ -12,7 +12,7 @@ import (
 )
 
 type FilesController struct {
-	baseController
+	baseController // nolint: unused
```

Comment:
not all those who wander are lost, and not all code that's unused should be deleted